### DISCUSSION TITLE
Prresent options on browser launch

### DISCUSSION THREAD

**Discussion by rebop** [sienori/Tab-Session-Manager#969] [Ideas]
Asked for this before, but would really like to open an browser and be asked which session to launch or None. The original Session Manager does this (and so much more) but for modern browsers TSM comes closest.

For example:

![image](https://user-images.githubusercontent.com/13785042/161165742-762cef4b-246d-4498-b512-b15226da625a.png)

While I am mentioning, Session Manager opens 50-100 tab sessions in maybe a 10th of the time TSM does if there is anything that can be done to speed that along.

Thanks so much.

~Bob

**Comment by rediffusion** [comment]
For me it's normal to open saved tabs after "browser launch".
(￣_,￣ )
